scripts/workers/code_upload_worker_utils/make_submission.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/challenge_phase/$PHASE_PK/submission/$SUBMISSION_PK"
```

Comment:
@savish28 where are the respective API changes to update the `submission_input_file` and republishing the submission message to the queue? For this API there is not PATCH update available

Reply:
@Ram81 `change_submission_data_and_visibility` in jobs views.py is related to this, we have made changes to send message in this particular view in a previous PR that is merged.